Jenkinsfile

```diff
@@ -53,7 +53,7 @@ pipeline {
 
               echo "Docker tag: ${env.DOCKER_TAG}"
               // build and push docker images
-              sh "mvn -Pci jib:build -pl apps/distributed,apps/distributed-es6-backport,apps/memory -X"
+              sh "mvn -Pci jib:build -pl apps/distributed,apps/memory -X"
```

Comment:
you can forget it, it will prevent conflict with https://github.com/linagora/tmail-backend/pull/874

Reply:
Yep I dropped the wip commit :)